### PR TITLE
Fix: modules verification bug bash

### PR DESCRIPTION
**Pull request by Poafs1**

### CODE REVIEW

File: src/lib/pages/modules-verify/components/ModuleVerifyModalBody.tsx [hunk 1]
```diff
@@ -19,8 +20,23 @@ export const ModuleVerifyModalBody = ({
...
+      setFakeLoading((newFakeLoading) => !newFakeLoading);
+    }, 1500);
+
+    return () => clearTimeout(timeoutId);
+  }, [isLoading]);
+
   if (isError) return <ModuleVerifyFailedModal onClose={onClose} />;
 
-  return <ModuleVerifySuccessModal onClose={onClose} control={control} />;
+  return fakeLoading ? (
```
Comment (songwongtp) [inline src/lib/pages/modules-verify/components/ModuleVerifyModalBody.tsx:37]: isLoading || fakeLoading

File: src/lib/pages/my-module-verification-details/index.tsx [hunk 2]
```diff
@@ -37,23 +38,49 @@ const MyModuleVerificationDetailsBody = ({ taskId }: { taskId: string }) => {
...
+      <EmptyState
+        imageVariant="not-found"
+        heading="Verification Details is Unavailable"
+        withBorder
+        message=""
+      >
+        <Box>
+          <Text color="text.dark">
+            We’re unable to access the page you requested due to one of the
+            following possibilities:
```
Comment (songwongtp) [inline src/lib/pages/my-module-verification-details/index.tsx:52]: one line

**Review by songwongtp — APPROVED** [review]

**Review by evilpeach — APPROVED** [review]
Fix as @songwongtp mentioned

File: src/lib/pages/my-module-verification-details/index.tsx (hunk 2 above)
Comment (Poafs1) [reply src/lib/pages/my-module-verification-details/index.tsx:52]: It's a linter. It will display in 1 line anyway.

**Pull request merged**